### PR TITLE
updated angular style guide

### PR DESCRIPTION
**Pull request by cbitstech**
add Application.js and examples of factory vs service module styles

### CODE REVIEW

**Comment by ericcf** [comment]
Please read the instructions for contributing in the top level README: https://github.com/cbitstech/guides. Thanks!

**Comment by cbitstech** [comment]
I've updated the branch and pull request, however line 3 still fails without a clear resolution, any ideas?

**Comment by ericcf** [comment]
It's not failing on your changes (nor for me locally), so I'm not too concerned.

File: development/angular/README.md [hunk 1]
```diff
@@ -17,15 +18,41 @@ contained therein.
     /Blue.js
 ```
 
+## Application.js
+
+The Application.js file should list out all `$routeProvider` route definitions
```
Comment (ericcf) [inline development/angular/README.md]: What do you think about "should configure application routes"?

File: development/angular/README.md [hunk 2]
```diff
@@ -17,15 +18,41 @@ contained therein.
...
+as well as including any `$rootScope` dependent application logic or constants.
```
Comment (ericcf) [inline development/angular/README.md]: This is not clear to me without an example.

File: development/angular/README.md [hunk 3]
```diff
@@ -17,15 +18,41 @@ contained therein.
...
+When using `$routeProvider` route definitions **must** use the `controllerAs` argument.
+
 ## Controllers
 
-JavaScript "classes" and non-Angular-specific code are always preferable to the
-alternative. With this in mind, all controllers **must** follow the alias
-pattern. E.g. when using `$routeProvider` route definitions **must** use the
-`controllerAs` argument.
+JavaScript "classes" and non-Angular-specific code are always preferable
+to thealternative. With this in mind, all controllers **must** follow the alias
```
Comment (ericcf) [inline development/angular/README.md]: typo

File: development/angular/README.md (hunk 1 above)
Comment (cbitstech) [inline development/angular/README.md]: I like it, updated.

File: development/angular/README.md (hunk 2 above)
Comment (cbitstech) [inline development/angular/README.md]: Indicates to me that it is not necessary. Removing.

File: development/angular/README.md (hunk 3 above)
Comment (cbitstech) [inline development/angular/README.md]: fixed

File: development/angular/README.md [hunk 4]
```diff
@@ -17,15 +18,40 @@ contained therein.
...
 
 Unless there is a reason that a Service must be used, Factories **should**
 always be favored over Services. This removes any temptation to write code that
 expects to always receive a particular object instance.
+
+```
+var myApp = angular.module('myApp', []);
+
+//factory style, USE THIS
+myApp.factory('helloWorldFromFactory', function() {
```
Comment (michaelwehrley) [inline development/angular/README.md:43]: could we change this to 'HellowWorldFromFactory'?

**Pull request merged**